mappings/translation/plan_general_characteristics.json

```diff
@@ -24,6 +24,24 @@
       ]
     }
   },
+  "existingModelID": {
+    "gqlField": "existingModelID",
+    "goField": "ExistingModelID",
+    "dbField": "existing_model_id",
+    "label": "What existing model is this a track of?",
+    "dataType": "NUMBER",
+    "formType": "TEXT",
+    "tableReference": "existing_model"
+  },
+  "currentModelPlanID": {
+    "gqlField": "currentModelPlanID",
+    "goField": "CurrentModelPlanID",
+    "dbField": "current_model_plan_id",
+    "label": "What model is this a track of?",
+    "dataType": "UUID",
+    "formType": "TEXT",
+    "tableReference": "model_plan"
+  },
```

Comment:
I think the merge overwrote the property "hideFromReadonly"

Reply:
Thanks for catching that Patrick! I'll update it

Reply:
It was there, was looking in the export, where it's not needed, my bad!

Reply:
No problem at all :)